packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -88,7 +88,9 @@ export function reduceField(options: ReduceFieldOptions): FieldCalcs {
     return (field.state.calcs = calcs);
   }
 
-  const { nullValueMode } = field.config;
+  // Default to Ignore for nullValueMode.
+  const { nullValueMode = NullValueMode.Ignore } = field.config;
```

Comment:
This change restores the previous behavior of defaulting to ignoring null values when we do calculate Count. 

But pretty confused by the fieldConfig.nullValueMode , we use it here but it's never set anywhere except in the reduce transformer. So for visualizations it will always be undefined (and here this means treated as ignore)